src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ProcessorService.cs

```diff
@@ -100,24 +92,19 @@ public async Task ProcessStage4Messages(ImportMessage message, ICollector<Import
         
         private async Task ProcessSubject(
             ImportMessage message,
-            StatisticsDbContext statisticsDbContext,
-            ContentDbContext contentDbContext,
             SubjectData subjectData)
         {
-            var subject = _releaseProcessorService.CreateOrUpdateRelease(subjectData,
-                message,
-                statisticsDbContext,
-                contentDbContext);
+            var statisticsDbContext = DbUtils.CreateStatisticsDbContext();
```

Comment:
Not seen this before, and it's not a limitation of using Azure Functions. Presumably it's is a quirk of the Importer, that every import needs a new `StatisticsDbContext`.

Reply:
Discussed on Slack - Duncan says:
```It's cos the processor runs many threads, and some of its dependencies are singletons. Cos they're singletons, they risk sharing the db context among 2 or more threads, which is a big no no for Entity Framework```